cmd/argo/commands/archive/list.go

```diff
@@ -47,28 +48,29 @@ func NewListCommand() *cobra.Command {
 				return err
 			}
 			namespace := client.Namespace()
-			workflows, err := listArchivedWorkflows(ctx, serviceClient, namespace, selector, chunkSize)
+			workflows, err := listArchivedWorkflows(ctx, serviceClient, namespace, selector, chunkSize, prefix)
 			if err != nil {
 				return err
 			}
 			return printer.PrintWorkflows(workflows, os.Stdout, printer.PrintOpts{Output: output.String(), Namespace: true, UID: true})
 		},
 	}
 	command.Flags().VarP(&output, "output", "o", "Output format. "+output.Usage())
+	command.Flags().StringVar(&prefix, "prefix", "", "Filter workflows by prefix")
```

Comment:
I feel like this should be more specific like "name prefix"; the API parameter is `NamePrefix` too, so it would be good to match
```suggestion
	command.Flags().StringVar(&prefix, "name-prefix", "", "Filter workflows by name prefix")
```

Reply:
It would also make sense to consider the style of #13160, where `--name` can be modified by `--name-filter`, which is one of "Exact" / "Prefix" / "Pattern".

Although it didn't implement that for the Archived API specifically

Reply:
I see `--prefix` and the description verbatim [match the `list` CLI](https://github.com/argoproj/argo-workflows/blob/bb5130e841ab02cfe5b65f19c214fe309ca88bce/cmd/argo/commands/list.go#L117), so perhaps we leave that in for backward-compat but also add `--name` and `--name-filter` to match #13160